### PR TITLE
Reintroduced StandardUserInterface to enable backwards compatibility.

### PR DESCRIPTION
**Pull request by AlexGilleran**

### CODE REVIEW

File: lib/ReactViews/Map/Panels/setting-panel.scss [hunk 1]
```diff
@@ -26,6 +26,9 @@
 .btn--viewer {
   composes: btn from '../../../Sass/common/_buttons.scss';
   composes: btn-grey from '../../../Sass/common/_buttons.scss';
+
+  color: $text-light;
```
Comment (RacingTadpole) [inline lib/ReactViews/Map/Panels/setting-panel.scss:30]: Just wondering if it's worth adding a "deprecated" comment to these added lines as well, so we know we can safely remove them again one day?

**Comment by RacingTadpole** [comment]
oop - found one (or it could be my own dodgy build)
image

**Comment by RacingTadpole** [comment]
oop - another one under ABS :-)
image

**Comment by RacingTadpole** [comment]
one more (looks like a few things in this screenshot though)...
image
http://localhost:3003/#build/terriajs/test/init/charts.json

File: lib/ReactViews/Map/Panels/setting-panel.scss (hunk 1 above)
Comment (AlexGilleran) [reply lib/ReactViews/Map/Panels/setting-panel.scss:30]: Unfortunately I don't think we'll ever be sure of that - I'm pretty sure a lot of these are actually just fixing stuff I missed too :(.

**Pull request merged**